test/core/test_promise.c

```diff
@@ -462,6 +462,42 @@ static em_promise_result_t test_any(void** result, void* data, void* value) {
   return EM_PROMISE_MATCH_RELEASE;
 }
 
+static em_promise_result_t test_race(void** result, void* data, void* value) {
+  emscripten_console_log("test_race");
+  assert(data == (void*)7);
+
+  em_promise_t in[2] = {emscripten_promise_create(),
```

Comment:
Not that it matters but I think you can drop the "2" here and below and have it inferred.   Might make the code less readable?

Reply:
Yeah, that's true. I kind of like having the explicit visual marker where I can check that the declared size matches the size I pass to the API function, but I don't think it matters too much.